SourceSinkPushPull/scripts/lib.lua

```diff
@@ -531,23 +531,27 @@ function set_haulers_to_manual(hauler_ids, message, item, stop)
     end
 end
 
+---@param stop LuaEntity
+function create_direct_to_station_order(stop)
+    -- Function from Cybersyn
+    local conditions_direct_to_station = { { type = "time", compare_type = "and", ticks = 1 } }
+    return {
+        rail = stop.connected_rail,
+        rail_direction = stop.connected_rail_direction,
+        wait_conditions = conditions_direct_to_station,
+    }
+end
+
```

Comment:
Is the wait condition needed here? I think since a recent factorio version it shouldn't be needed anymore (I'm already using an order without a condition when arriving at stations)

Also, this is only used in one place, so I would prefer it just be inline in the send_hauler_to_* function